CHANGELOG.md

```diff
@@ -1,3 +1,274 @@
+- Tabulate buffer picker contents (#5777)
+- Add an option to disable LSP (#4425)
+- Short-circuit tree-sitter and word object motions (#5851)
+- Add exit code to failed command message (#5898)
+- Make `m` textobject look for pairs enclosing selections (#3344)
+- Negotiate LSP position encoding (#5894)
+- Display deprecated LSP completions with strikethrough (#5932)
+- Add JSONRPC request ID to failed LSP/DAP request log messages (#6010, #6018)
+- Ignore case when filtering LSP completions (#6008)
+- Show current language when no arguments are passed to `:set-language` (#5895)
+- Refactor and rewrite all book documentation (#5534)
+- Check LSP renaming support before prompting for rename text (#6257)
+- Separate diagnostic picker message and code (#6095)
+- Add a config option to bypass undercurl detection (#6253)
+- Only complete appropriate arguments for typed commands (#5966)
+- Discard outdated LSP diagnostics (3c9d5d0)
+- Discard outdated LSP workspace edits (b6a4927)
+- Run shell commands asynchronously (#6373)
+- Show diagnostic codes in LSP diagnostic messages (#6378)
+
+Fixes:
+
+- Fix behavior of `auto-completion` flag for completion-on-trigger (#5042)
+- Reset editor mode when changing buffers (#5072)
+- Respect scrolloff settings in mouse movements (#5255)
+- Avoid trailing `s` when only one file is opened (#5189)
+- Fix erroneous indent between closers of auto-pairs (#5330)
+- Expand `~` when parsing file paths in `:open` (#5329)
+- Fix theme inheritance for default themes (#5218)
+- Fix `extend_line` with a count when the current line(s) are selected (#5288)
+- Prompt: Fix autocompletion for paths containing periods (#5175)
+- Skip serializing JSONRPC params if params is null (#5471)
+- Fix interaction with the `xclip` clipboard provider (#5426)
+- Fix undo/redo execution from the command palette (#5294)
+- Fix highlighting of non-bar cursors (#5575)
+- Fix panic when nooping in `join_selections` and `join_selections_space` (#5423)
+- Fix selecting a changed file in global search (#5639)
+- Fix initial syntax highlight layer sort order (#5196)
+- Fix UTF-8 length handling for shellwords (#5738)
+- Remove C-j and C-k bindings from the completion menu (#5070)
+- Always commit to history when pasting (#5790)
+- Properly handle LSP position encoding (#5711)
+- Fix infinite loop in `copy_selection_on_prev_line` (#5888)
+- Fix completion popup positioning (#5842)
+- Fix a panic when uncommenting a line with only a comment token (#5933)
+- Fix panic in `goto_window_center` at EOF (#5987)
+- Ignore invalid file URIs sent by a language server (#6000)
+- Decode LSP URIs for the workspace diagnostics picker (#6016)
+- Fix incorrect usages of `tab_width` with `indent_width` (#5918)
+- DAP: Send Disconnect if the Terminated event is received (#5532)
+- DAP: Validate key and index exist when requesting variables (#5628)
+- Fix indent guide rendering (#6136)
+- Fix division by zero panic (#6155)
+- Fix scrolloff at view bottom (#6142)
```

Comment:
I out could include this with softwrap above since this is a fix for a regression frk #5420 and not an existing bug